frontend/src/domain/Template.ts

```diff
@@ -1,9 +1,9 @@
 export type Template = {
   id: number
   name: string
-  story: string
-  description: string
-  taskType: string
+  story: string | null
+  description: string | null
+  taskType: string | null
   userId: number
   projectId: number
```

Comment:
The projectId and userId are also nullable in a template. When we start using global templates in the list, the userId will be null for those. A user's personal templates will have their userId.